src/app/teams/teams-view.component.ts

```diff
@@ -215,10 +215,13 @@ export class TeamsViewComponent implements OnInit, OnDestroy {
   }
 
   changeMembership(type, memberDoc?) {
-    this.changeMembershipRequest(type, memberDoc).subscribe((message) => {
-      this.setStatus(this.team, this.userService.get());
-      this.planetMessageService.showMessage(message);
-    });
+    return forkJoin([
```

Comment:
`forkJoin` should have multiple database requests in the array.  It's best used in the context of our app to make multiple, independent requests to the database.